src/QMCWaveFunctions/RotatedSPOs.h

```diff
@@ -203,28 +226,15 @@ class RotatedSPOs : public SPOSet, public OptimizableObject
 
   void checkInVariablesExclusive(opt_variables_type& active) override
   {
-    //reset parameters to zero after coefficient matrix has been updated
-    for (int k = 0; k < myVars.size(); ++k)
-      myVars[k] = 0.0;
-
+    use_this_copy_to_apply_rotation_ = true;
     if (myVars.size())
       active.insertFrom(myVars);
-    Phi->storeParamsBeforeRotation();
   }
 
   void checkOutVariables(const opt_variables_type& active) override { myVars.getIndex(active); }
 
   ///reset
-  void resetParametersExclusive(const opt_variables_type& active) override
-  {
-    std::vector<RealType> param(m_act_rot_inds.size());
-    for (int i = 0; i < m_act_rot_inds.size(); i++)
-    {
-      int loc  = myVars.where(i);
-      param[i] = myVars[i] = active[loc];
-    }
-    apply_rotation(param, true);
-  }
+  void resetParametersExclusive(const opt_variables_type& active) override;
```

Comment:
So I guess this touches something that shares data with other threads/spos?  Could use some documentation.  Otherwise I just don't know what exclusive means.

Reply:
It's documented in OptimizableObject.h.

Reply:
Ok I'm was just confused by exclusive not having read that recently.